include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -1534,6 +1559,12 @@ class FEEvaluationAccess<dim, 1, Number, is_face, VectorizedArrayType>
   void
   submit_gradient(const gradient_type grad_in, const unsigned int q_point);
 
+  /**
+   * @copydoc FEEvaluationBase<dim,1,Number,is_face>::submit_hessian()
+   */
+  void
+  submit_hessian(const hessian_type hessian_in, const unsigned int q_point);
+
```

Comment:
Can you move this down to line 1580 below the `get_hessian_diagonal` function, to have similar things grouped together. While there, can you also change the return type of `get_hessian` to `hessian_type`, to make use of the new typedef and create consistent names?